ansible/kubernetes/roles/ingresses/tasks/main.yml

```diff
@@ -42,6 +42,34 @@
       path: /terrain
       service: terrain
 
+- name: Add agave-callback ingress
+  delegate_to: localhost
+  environment:
+    KUBECONFIG: "{{ lookup('env', 'KUBECONFIG') }}"
+  kubernetes.core.k8s:
+    state: present
+    definition:
+      apiVersion: networking.k8s.io/v1
+      kind: Ingress
+      metadata:
+        name: agave-callback
+        namespace: "{{ ns }}"
+        annotations:
+          traefik.ingress.kubernetes.io/rewrite-target: /callbacks/agave-job
+      spec:
+        ingressClass: traefik
+        rules:
+          - host: de.cyverse.org
```

Comment:
I wonder if we should add a variable for the host so that we don't have to configure HAProxy to always use `de.cyverse.org` for the host name. If we do that, then I _think_ that the header will automatically be populated.

Reply:
Good call, I'll get it fixed.

Reply:
Okay, we should have a var named `de_hostname` now with a default value of `de.cyverse.org`.